html/Corporate/style.scss

```diff
@@ -0,0 +1,664 @@
+@import url("https://fonts.googleapis.com/css?family=Exo:300,400,700&display=swap");
```

Comment:
using scss you can split you files(by blocks, functionality, components anything what have sense for certain mockup) (here you have comments by section it is can be good separation to files) and import to one, it is much easier to work with a few small files than with one big file